cms/static/coffee/spec/main.coffee

```diff
@@ -221,8 +221,11 @@ define([
     "coffee/spec/models/settings_grading_spec", "coffee/spec/models/textbook_spec",
     "coffee/spec/models/upload_spec",
 
-    "coffee/spec/views/metadata_edit_spec", "coffee/spec/views/module_edit_spec",
-    "coffee/spec/views/textbook_spec", "coffee/spec/views/upload_spec",
+    "coffee/spec/views/course_info_spec",
```

Comment:
Looking at that PR (https://github.com/edx/edx-platform/pull/9265/files#diff-ba518c58b88bfebd8cf4d5941284d747L220), do we need to add those other spec files back in as well?